src/brevitas_examples/bnn_pynq/models/resnet.py

```diff
@@ -39,10 +45,11 @@ def __init__(
             in_planes,
             planes,
             stride=1,
+            bias=False,
             shared_quant_act=None,
-            weight_bit_width=8,
             act_bit_width=8,
-            bias=False):
+            weight_bit_width=8,
+            weight_quant=Int8WeightPerTensorFloat):
```

Comment:
You can set as default Int8WeightPerChannelFloat and get rid of the weight_scaling_per_output_channel=True set explicitly